scripts/pre_commit_linter.py

```diff
@@ -0,0 +1,153 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Pre-commit script for oppia
+This script checks Javascript coding style using jscs node module
+It returns and fix errors
```

Comment:
Does it actually fix errors? I thought we talked about passing in an option to do this (using e.g. argparse), but I don't see that here.